app/code/community/Ebizmarts/MailChimp/Helper/Data.php

```diff
@@ -1688,13 +1688,14 @@ protected function getCoreWebsite()
      * @param  $subscriberEmail
      * @return null
      */
-    public function getLastDateOfPurchase($subscriberEmail)
+    public function getLastDateOfPurchase($subscriberEmail, $lastOrder = null)
     {
         $lastDateOfPurchase = null;
+        if (!$lastOrder) {
```

Comment:
Use exact operator $lastOrder === null to avoid false positives in case 0 or '' is given.